packages/jellyfish-transaction-builder/__tests__/txn/varint_gt_128.test.ts

```diff
@@ -599,56 +599,58 @@ it('should setCollateralToken, takeLoan and paybackLoan', async () => {
   await providers.setupMocks()
   const script = await providers.elliptic.script()
 
+  // not allowed for non-DAT token
   // test DCT token on `setCollateralToken`
-  {
-    await fundEllipticPair(container, providers.ellipticPair, 10)
-
-    const txn = await builder.loans.setCollateralToken({
-      token: 128,
-      factor: new BigNumber(1),
-      currencyPair: { token: 'FISH#128', currency: 'USD' },
-      activateAfterBlock: 0
-    }, script)
-
-    // Ensure the created txn is correct.
-    const outs = await sendTransaction(container, txn)
-    expect(outs[0].value).toStrictEqual(0)
-
-    const col = await testing.rpc.loan.getCollateralToken('FISH#128')
-    expect(col).toStrictEqual({
-      token: 'FISH#128',
-      tokenId: expect.stringMatching(/[0-f]{64}/),
-      factor: new BigNumber(1),
-      fixedIntervalPriceId: 'FISH#128/USD'
-    })
-  }
-
+  // {
+  //   await fundEllipticPair(container, providers.ellipticPair, 10)
+
+  //   const txn = await builder.loans.setCollateralToken({
+  //     token: 128,
+  //     factor: new BigNumber(1),
+  //     currencyPair: { token: 'FISH#128', currency: 'USD' },
+  //     activateAfterBlock: 0
+  //   }, script)
+
+  //   // Ensure the created txn is correct.
+  //   const outs = await sendTransaction(container, txn)
+  //   expect(outs[0].value).toStrictEqual(0)
+
+  //   const col = await testing.rpc.loan.getCollateralToken('FISH#128')
+  //   expect(col).toStrictEqual({
+  //     token: 'FISH#128',
+  //     tokenId: expect.stringMatching(/[0-f]{64}/),
+  //     factor: new BigNumber(1),
+  //     fixedIntervalPriceId: 'FISH#128/USD'
+  //   })
+  // }
+
+  // not allowed for non-DAT token
```

Comment:
What happened to this?

Reply:
https://github.com/DeFiCh/ain/blob/ea7b1806babc3af760df30b136b34c996d67b29f/src/masternodes/govvariables/attributes.cpp#L1776-L1784

hmm.. its prev bug.. as non DATs should not be set as collateral tokens in the native DEX

fixes at:
https://github.com/DeFiCh/ain/pull/2258#discussion_r1287555257